x/protorev/types/events.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+const (
+	TypeEvtBackrun = "protorev_backrun"
+
+	AttributeValueCategory               = ModuleName
```

Comment:
I think it would be a nice addition to also emit the hash of the transaction that is being backrun.
```suggestion
	AttributeValueCategory               = ModuleName
    AttributeKeyTransactionHash          
```suggestion
	AttributeValueCategory               = ModuleName
```= "tx_hash"
```

Reply:
added in e89701b9b3c6e2638a341f3c41364e94447783a7